spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebQuteReactiveIT.java

```diff
@@ -18,8 +18,10 @@ public class SpringWebQuteReactiveIT extends AbstractSpringWebQuteReactiveIT {
             .withProperty("quarkus.datasource.username", database.getUser())
             .withProperty("quarkus.datasource.password", database.getPassword())
             .withProperty("quarkus.datasource.reactive.url",
-                    () -> "vertx-reactive:" + database.getHost().replace("http", "mysql") + ":" + database.getPort() + "/"
-                            + database.getDatabase());
+                    () -> "vertx-reactive:" + database.getURI()
```

Comment:
This feels like a (verbose) pattern, what about (also not optimal)
`"quarkus.datasource.reactive.url", database::getReactiveUrl`
and 
```Java
static final MariaDbService database = new MariaDbService() {
        @Override
        protected String getJdbcName() {
            return "mysql";
        }
    };
```

Reply:
Hmm, it works also without `@Override` with MariaDB, what am I doing wrong?

Reply:
Okay, this test is actually not testing database but it is starting it :-D

Reply:
@gtroitsk please remove MariaDB container from this test, it passes anyway.